### PR TITLE
Udev opts tweaks

### PR DESCRIPTION
**Pull request by fbuihuu**

### CODE REVIEW

File: src/basic/meson.build [hunk 1]
```diff
@@ -329,6 +329,7 @@ libbasic = static_library(
         dependencies : [versiondep,
                         threads,
                         libcap,
+                        libseccomp,
```
Comment (keszybz) [inline src/basic/meson.build]: I'd very much prefer to move that code to src/shared/ and not introduce the dependency here.

Comment (poettering) [reply src/basic/meson.build]: yupp, this is a bug. the relevant functions should be moved to src/shared/

@topimiettinen

**Review by poettering — CHANGES_REQUESTED** [review]

**Comment by fbuihuu** [comment]
OK dropped the offending commit for now.

**Comment by keszybz** [comment]
centosci's are failing in TEST-50-DISSECT, looks unrelated.

**Pull request merged**